pylint_odoo/checkers/itp_checks.py

```diff
@@ -0,0 +1,38 @@
+import re
+import ast
+import os
+import types
+import astroid
+from pylint.checkers import BaseChecker, utils
+from pylint.interfaces import IAstroidChecker
+from .. import settings
+
+ITP_CHECK_PREFIX_ID = 77
+ITP_ODOO_MSGS = {
+    # C->convention R->refactor W->warning E->error F->fatal
+    'E%d01' % ITP_CHECK_PREFIX_ID: (
+        'Necessary manifest field "%s" got wrong value "%s"',
+        'skipped-field',
+        settings.DESC_DFLT
+    ),
+}
+
+
+class ITPModuleChecker(BaseChecker):
+    __implements__ = IAstroidChecker
+
+    name = settings.CFG_SECTION
+    msgs = ITP_ODOO_MSGS
+
+    def visit_dict(self, node):
+        if not os.path.basename(self.linter.current_file) in settings.MANIFEST_FILES \
+                or not isinstance(node.parent, astroid.Discard):
+            return
```

Comment:
Are you sure, that this check is applied for all manifist and doc files ? 

README.rst
doc/index.rst
openerp.py / manifest.py
doc/changelog.rst